lib/licenses.nix

```diff
@@ -481,6 +481,10 @@ in mkLicense lset) ({
     free = false;
   };
 
+  fraunhofer-fdk = {
+    fullName = "Software License for The Fraunhofer FDK AAC Codec Library";
```

Comment:
Looks great!

I think you can add a known `spdxId`, and probably use full name from there as well:: https://spdx.org/licenses/FDK-AAC.html . Maybe something like:

```nix
spdxId = "FDK-AAC";
fullName = "Fraunhofer FDK AAC Codec Library";
```

Reply:
Done, though i'm not sure if adding a new commit is the best way to do this or if i should rewrite the commit to have had the correct info in the first place...